README.md

```diff
@@ -31,7 +31,7 @@ random selection of the reliable nodes it knows about.
 To build and install from a checked-out repo, run `go install` in the repo's
 root directory.
 
-To start dcrseeder listening on udp 127.0.0.1:5354 with an initial connection to working testnet node 192.168.0.1:
+To start dcrseeder listening on udp localhost:8000 with an initial connection to working testnet node 192.168.0.1:
```

Comment:
hmm, this is tcp now too, right?

Reply:
lol yeah just  commented that. since its http its tcp